.env.example

```
@@ -0,0 +1,42 @@
+# Admin access
+ADMIN_EMAILS=admin@example.com,owner@example.com
+
+# Google OAuth
+GOOGLE_CLIENT_ID=your_google_client_id
+GOOGLE_CLIENT_SECRET=your_google_client_secret
+
+# Resend Email Service
+RESEND_API_KEY=re_123456789
+# Optional: The email address that will appear in the "From" field.
+# Must be a verified domain in Resend. If not set, it defaults to onboarding@resend.dev
+EMAIL_FROM=Payzone <no-reply@payzone.store>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_FROM key should go before the RESEND_API_KEY key

(UnorderedKey)